Classes/IndexQueue/RecordMonitor.php

```diff
@@ -45,13 +43,6 @@ class RecordMonitor extends AbstractDataHandlerListener
 {
 
     /**
-     * Solr TypoScript configuration
-     *
-     * @var TypoScriptConfiguration
-     */
-    protected $solrConfiguration;
```

Comment:
Will this have any impact? / Why are you removing this? Since now you have to hand the configuration to all these methods.